lib/shared/permissions.js

```diff
@@ -198,6 +198,13 @@ exports.elevateCommand = (command, options) => {
   // for now, we should make sure we double check if the error messages
   // have changed every time we upgrade `sudo-prompt`.
 
+  }).catch((error) => {
+    return _.includes(error.message, 'is not in the sudoers file');
+  }, () => {
+    throw errors.createUserError({
+      title: 'Your user doesn\'t have enough permissions to proceed',
```

Comment:
This line should also be using `privileges` - sorry if that wasn't clear enough in my last comment.